tests/api/test_dnf_module_package.py

```diff
@@ -0,0 +1,134 @@
+
+    def tearDown(self):
+        self.base.close()
+
+    def test_getName(self):
+        # ModulePackage.getName()
+        self.assertHasAttr(self.modulePackage, "getName")
+        self.modulePackage.getName()
+
+    def test_getStream(self):
+        # ModulePackage.getStream()
+        self.assertHasAttr(self.modulePackage, "getStream")
+        self.modulePackage.getStream()
+
+    def test_getVersion(self):
+        # ModulePackage.getVersion()
+        self.assertHasAttr(self.modulePackage, "getVersion")
+        self.modulePackage.getVersion()
+
+    def test_getVersionNum(self):
+        # ModulePackage.getVersionNum()
+        self.assertHasAttr(self.modulePackage, "getVersionNum")
+        self.modulePackage.getVersionNum()
+
+    def test_getContext(self):
+        # ModulePackage.getContext()
+        self.assertHasAttr(self.modulePackage, "getContext")
+        self.modulePackage.getContext()
+
+    def test_getArch(self):
+        # ModulePackage.getArch()
+        self.assertHasAttr(self.modulePackage, "getArch")
+        self.modulePackage.getArch()
+
+    def test_getNameStream(self):
+        # ModulePackage.getNameStream()
+        self.assertHasAttr(self.modulePackage, "getNameStream")
+        self.modulePackage.getNameStream()
+
+    def test_getNameStreamVersion(self):
+        # ModulePackage.getNameStreamVersion()
+        self.assertHasAttr(self.modulePackage, "getNameStreamVersion")
+        self.modulePackage.getNameStreamVersion()
+
+    def test_getFullIdentifier(self):
+        # ModulePackage.getFullIdentifier()
+        self.assertHasAttr(self.modulePackage, "getFullIdentifier")
+        self.modulePackage.getFullIdentifier()
+
+    def test_getProfiles(self):
+        # ModulePackage.getProfiles()
+        self.assertHasAttr(self.modulePackage, "getProfiles")
+        self.modulePackage.getProfiles()
+
+    def test_getSummary(self):
+        # ModulePackage.getSummary()
+        self.assertHasAttr(self.modulePackage, "getSummary")
+        self.modulePackage.getSummary()
+
+    def test_getDescription(self):
+        # ModulePackage.getDescription()
+        self.assertHasAttr(self.modulePackage, "getDescription")
+        self.modulePackage.getDescription()
+
+    def test_getRepoID(self):
+        # ModulePackage.getRepoID()
+        self.assertHasAttr(self.modulePackage, "getRepoID")
+        self.modulePackage.getRepoID()
+
+    def test_getArtifacts(self):
+        # ModulePackage.getArtifacts()
+        self.assertHasAttr(self.modulePackage, "getArtifacts")
+        self.modulePackage.getArtifacts()
+
+    def test_getModuleDependencies(self):
+        # ModulePackage.getModuleDependencies()
+        self.assertHasAttr(self.modulePackage, "getModuleDependencies")
+        self.modulePackage.getModuleDependencies()
+
+    def test_getYaml(self):
+        # ModulePackage.getYaml()
+        self.assertHasAttr(self.modulePackage, "getYaml")
+        self.modulePackage.getYaml()
+
+    # ModuleProfile
```

Comment:
How about to add a separate class for ModuleProfile tests? The same for ModuleDependencies.